PySDM/backends/impl_numba/methods/condensation_methods.py

```diff
@@ -405,15 +405,13 @@ def minfun(  # pylint: disable=too-many-arguments,too-many-locals
                     temperature, volume_new, formulae.constants.PI_4_3 * rd3, f_org
                 ),
             )
-            r_dr_dt = formulae.drop_growth__r_dr_dt(
-                RH_eq,
-                temperature,
-                RH,
-                lv,
-                pvs,
-                D * mass_ventilation_factor,
-                K * heat_ventilation_factor,
+            Fk = formulae.drop_growth__Fk(
```

Comment:
we are not updating the ventilation coefficients here (but we probably should...), so recomputing Fk and Fd is not needed (yet...)